R/checkInputs.R

```diff
@@ -246,3 +246,31 @@ correctCdmFormat <- function(table, tableName) {
 
 
 }
+
+#' get occurrence information from cdm table
+#'
+#' @noRd
+#'
+getOccurrence <- function(cdm, tableName, start, end) {
+
+  dataTable <- cdm[[tableName]] |>
+    dplyr::select("person_id", start, end) |>
+    dplyr::group_by() |>
+    dplyr::rename(visit_start_date = start,
```

Comment:
you can do the rename at the select stage

Reply:
this function is not needed